packages/default-theme/components/forms/SwEmail.vue

```diff
@@ -67,44 +67,55 @@ import { validationMixin } from 'vuelidate'
 import { required, email, sameAs } from 'vuelidate/lib/validators'
 import { SfInput, SfButton, SfAlert } from '@storefront-ui/vue'
 import { useUser } from '@shopware-pwa/composables'
+import { getMessagesFromErrorsArray } from '@shopware-pwa/helpers'
 
 export default {
-  name: 'MyProfile',
+  name: 'EmailChange',
   components: { SfInput, SfButton, SfAlert },
   mixins: [validationMixin],
   props: {},
   setup() {
-    const { user, error, updateEmail, refreshUser } = useUser()
+    const { user, error: userError, updateEmail, refreshUser } = useUser()
```

Comment:
we can do computation here, getMessagesFromErrorsArray checks for nulls so we can pass just argument :)

```suggestion
    const { user, error: userError, updateEmail, refreshUser } = useUser()
    const userErrorMessages = computed(() => getMessagesFromErrorsArray(userError.value.message))
```